src/tileset.cpp

```diff
@@ -28,6 +28,7 @@
 #include <QRadioButton>
 #include <QSettings>
 #include <QToolButton>
+#include <QRubberBand>
```

Comment:
This was causing compilation errors.

Reply:
Really? What kind of compilation error. QRubberBand is part of widgets, header should be there (though it's fine to remove it also, doesn't look like anything in tileset.cpp uses that class anyway).

Reply:
Without this line, I got an error about "forward declaration" of a class, it refused to build. The specific error was in this line, at `QRubberBand::Rectangle`:
```mRubberBand = new QRubberBand(QRubberBand::Rectangle, this);```

Reply:
Ah, I see. QRubberBand was likely included by another header we already include in previous Qt releases but isn't anymore. Makes sense, cool.